pages/blog/[slug].js

```diff
@@ -0,0 +1,44 @@
+import Head from 'next/head'
+import Layout from '../../components/layout'
+import BlogHeader from '../../components/elements/BlogHeader'
+import { getPosts } from '../api/posts'
+
+import RecoverBlog from '../api/posts/recover'
+import EscrowBlog from '../api/posts/escrow'
+
+const blogsMap = {
+  'lost-and-found-iphone': RecoverBlog,
+  'securing-valuables-with-escrow-smart-contracts': EscrowBlog
+}
```

Comment:
We don't need this mapping.
The HTML content has to be in the `../api/posts/index.js`.

But you can create a different file like you didi it and import in this file and add an attribute like this: `content: htmlContentEscrow`.

Reply:
@bhavesh92pandey @n1c01a5 Next.js doesn't allow JSX elements to be exported inside an object. So importing `htmlContentEscrow` and using it inside a `post` object in the `../api/posts/index.js` like `content: htmlContentEscrow` is not working. I'll try to figure out a non-complex way of doing it. I'll keep you posted about the progress.

Reply:
@amitg01 I think you can insert html with `dangerouslySetInnerHTML`.

Reply:
@n1c01a5 The problem here is with the storage of the content and not the usage. Like I mentioned earlier, Next.js does not allow something like `content: htmlContentEscrow` in an individual post object in `../api/posts/index.js`, here `htmlContentEscrow` is a jsx element. 

If it was allowed, I could have used `dangerouslySetInnerHTML` and set the content after the post object had been loaded.

We can schedule a call to discuss this if there is some doubt.

Reply:
Ok I will do some tests and I will answer you.

Reply:
Cool.

Reply:
See my code under the comment.